spec/integration/alerts_spec.rb

```diff
@@ -286,6 +286,13 @@ module Hawkular::Alerts::RSpec
       expect(events).to_not be_nil
       expect(events.size).to be(0)
     end
+
+    it 'Should create an event' do
```

Comment:
I would like to leave it as is. And then in a followup PR we should change this for all of the Alert/.. categories that do not have it yet.